CRM/Contribute/BAO/Contribution.php

```diff
@@ -4477,16 +4476,6 @@ public static function completeOrder($input, $ids, $objects, $isPostPaymentCreat
     $recurringContributionID = (empty($recurContrib->id)) ? NULL : $recurContrib->id;
     $event = $objects['event'] ?? NULL;
 
-    $paymentProcessorId = '';
```

Comment:
If we go wit https://github.com/civicrm/civicrm-core/pull/18315/files as a next step then this would be in $ids